test/fakes.ts

```diff
@@ -89,7 +90,13 @@ function fakeShellExec(settings : FakeShellSettings, cmd : string) : Promise<She
 
 export function kubectl(settings : FakeKubectlSettings = {}) : any {
     const asLines = settings.asLines || ((s : string) => []);
+    const invokeAsync = settings.invokeAsync || ((s : string) => ({ code: 0, stderr: "", stdout: ""}));
     return {
-        asLines: (cmd) => new Promise((resolve, reject) => resolve(asLines(cmd)))
+        asLines: (cmd) => wrapAsPromise(asLines(cmd)),
```

Comment:
wrapAsPromise doesn't need to wrapAsPromise, add async before (cmd)

Reply:
nice.